cniconf/watcher.go

```diff
@@ -78,7 +83,39 @@ func (w *watcher) onChange(version string) error {
 	return nil
 }
 
-func (w *watcher) apply(network metadata.Network) error {
+func (w *watcher) inserthostSubnetRules(host metadata.Host, content []byte) ([]byte, error) {
+	subnet, _ := host.Labels[hostSubnetLabel]
```

Comment:
@ibuildthecloud There are few options here:
1) Use another label to identify `per_host_subnet` network type say `io.rancher.network.per_host_subnet: true` or
2) Add the check based on metadata/network/name == "per-host-subnet" or
3) Add it in the CNI config/name: "rancher-per-host-subnet-cni-network"

Which one to pick?